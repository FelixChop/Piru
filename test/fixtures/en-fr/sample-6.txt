The CEO's decision to downsize was a bitter pill to swallow, especially after the company's profits had skyrocketed.
