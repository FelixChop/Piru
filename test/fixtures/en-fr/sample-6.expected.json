[
  {
    "word": "downsize",
    "lemma": "to downsize",
    "definition": "réduire les effectifs",
    "citation": "The CEO's decision to downsize was a bitter pill to swallow, especially after the company's profits had skyrocketed.",
    "to_learn": true
  },
  {
    "word": "bitter pill to swallow",
    "lemma": "bitter pill to swallow",
    "definition": "chose difficile à accepter",
    "citation": "The CEO's decision to downsize was a bitter pill to swallow, especially after the company's profits had skyrocketed.",
    "to_learn": true
  },
  {
    "word": "skyrocketed",
    "lemma": "to skyrocket",
    "definition": "monter en flèche",
    "citation": "The CEO's decision to downsize was a bitter pill to swallow, especially after the company's profits had skyrocketed.",
    "to_learn": true
  }
]
